aot/src/main/java/com/dylibso/chicory/experimental/aot/AotUtil.java

```diff
@@ -140,8 +140,14 @@ public static MethodType methodTypeFor(FunctionType type) {
         return rawMethodTypeFor(type).appendParameterTypes(Memory.class, Instance.class);
     }
 
+    public static boolean hasTooManyParameters(FunctionType type) {
+        return type.params().stream().mapToInt(AotUtil::slotCount).sum() > 253;
```

Comment:
nitpick: please define a static value for `253` with the rationale in a comment.

Reply:
done.

Reply:
thanks!